src/CitizenProject/CitizenProjectManager.php

```diff
@@ -344,6 +344,26 @@ public function removeImage(CitizenProject $citizenProject): void
         $citizenProject->setImageUploaded(false);
     }
 
+    public function changePrivilege(CitizenProject $citizenProject, Adherent $adherent, string $privilege, bool $flush = true): void
```

Comment:
you change the privilege of an adherent, for me adherent must be in the first position (+ for being as others `changePrivilege` methods of this project ;)